src/libs/highlevel/README.md

```diff
@@ -191,6 +216,9 @@ to use these functions directly, but they might still be useful sometimes (e.g.
 `elektraGetRawString`). We also provide a `KDB_TPYE_*` constant for each of the types listed above. Again, most users won't use these
 but, if you ever do need to use the raw type metadata using constants enables code completion and protects against typos.
 
+There is also the type `enum` with constant `KDB_TYPE_ENUM`. It is currently neither used nor supported by this API. However, we reserve it
+for a future expansion of this API.
```

Comment:
But it is possible to get the string, isn't it?

Reply:
You either have to set `type = string` and use `elektraGetString` or use `elektraGetRawString`. Both are compatible with the `enum` plugin because that only requires `check/enum`.

I will add a note about that.